src/lib/moonforge/index.js

```
@@ -0,0 +1,149 @@
+ * Wires up re-armable foreground session spans. Unlike a single session
+ * that ends permanently on the first tab-blur, this tracks each foreground
+ * span independently: a new 'session_start' fires whenever the tab becomes
+ * visible again, and a 'session_end' fires whenever it becomes hidden or
+ * the page is torn down, so total foreground duration stays accurate
+ * across repeated tab switches.
+ */
+let sessionListenersInstalled = false;
+
+function startSession() {
+  beginSpan();
+  if (sessionListenersInstalled || typeof globalThis.addEventListener !== 'function') return;
```

ESLint

[error] Replace `sessionListenersInstalled·||·typeof·globalThis.addEventListener·!==·'function')` with `⏎····sessionListenersInstalled·||⏎····typeof·globalThis.addEventListener·!==·"function"⏎··)⏎···`

(prettier/prettier)